plugins/scripts/inject-config.js

```diff
@@ -83,8 +83,44 @@ module.exports = function injectOptimoveConfig(context) {
         console.info('Optimove: Preparing Android platform...');
         prepareAndroid(context, optimoveConfig);
     }
+
+    if (hasPlatform(context, 'ios')) {
+        console.info('Optimove: Preparing iOS platform...');
+        prepareIos(context, optimoveConfig);
+    }
 };
 
+function prepareIos(context, kumulosConfig) {
+    const iosPath = path.join(context.opts.projectRoot, 'platforms', 'ios');
+    const files = fs.readdirSync(iosPath);
+
+    const xcodeProj = files.find(name => name.indexOf('.xcodeproj') > -1);
+    const targetName = xcodeProj.replace('.xcodeproj', '');
+
+    const configDest = path.join(
+        iosPath,
+        targetName,
+        'Resources',
+        'optimove.plist'
+    );
+
+    if (!fs.existsSync(configDest)) {
+        console.error(
+            'optimove: optimove.plist resource not found, aborting setup'
+        );
+        return;
+    }
+
+    const config = renderTemplate('optimove.plist', {
+        API_KEY: kumulosConfig.apiKey,
```

Comment:
OK think this looks good, just need to update the values to match the expected optimove / mobile credentials keys expected by the SDK for initialisation

Reply:
Done